esignet-default.properties

```diff
@@ -409,21 +409,23 @@ mosip.esignet.ui.config.key-values={'sbi.env': 'Developer', 'sbi.timeout.DISC':
   'sbi.timeout.DINFO': 30, 'sbi.timeout.CAPTURE': 30, 'sbi.capture.count.face': 1, 'sbi.capture.count.finger': 1, \
   'sbi.capture.count.iris': 1, 'sbi.capture.score.face': 70, 'sbi.capture.score.finger':70, 'sbi.capture.score.iris':70, \
   'resend.otp.delay.secs': ${mosip.kernel.otp.expiry-time}, 'send.otp.channels' : '${mosip.esignet.authenticator.ida.otp-channels}', \
-  'captcha.sitekey' : '${mosip.esignet.captcha-validator.site-key}', 'captcha.enable' : 'pwd', \
+  'captcha.sitekey' : '${mosip.esignet.captcha-validator.site-key}', 'captcha.enable' : '${mosip.esignet.captcha.required}', \
   'auth.txnid.length' : '${mosip.esignet.auth-txn-id-length}', 'consent.screen.timeout-in-secs':${mosip.esignet.authentication-expire-in-secs}, \
   'consent.screen.timeout-buffer-in-secs': 5, 'linked-transaction-expire-in-secs': 240, 'sbi.port.range': '4501-4600', \
   'sbi.bio.subtypes.iris': 'UNKNOWN', 'sbi.bio.subtypes.finger': 'UNKNOWN', 'wallet.qr-code-buffer-in-secs': 10, 'otp.length': 6, \'password.regex': '^.{8,20}$', \
   'password.max-length': 20, \
-  'username.regex': '^[0-9]{10,30}$',\
-  'username.prefix': '', \
-  'username.postfix': '', \
-  'username.max-length': 30, \
+  'username.regex': '^[1-9][0-9]{7,8}$',\
```

Comment:
For QA-inji env is not camdgc env so we should use default config
'username.regex': '^[0-9]{10,30}$',\
  'username.prefix': '', \
  'username.postfix': '', \
  'username.max-length': 30, \